### ISSUE TITLE
Errors:     error #000: ../../src/H5F.c in H5Fopen(): line 620: unable to open file         class: HDF5         major: File accessibility         minor: Unable to open file

### ISSUE THREAD

**Issue by wyj20031116** [mojaveazure/seurat-disk#188]
Convert("BLCA.h5Seurat", dest="h5ad")
Warning: HDF5-API Errors:
    error #000: ../../src/H5F.c in H5Fopen(): line 620: unable to open file
        class: HDF5
        major: File accessibility
        minor: Unable to open file

    error #001: ../../src/H5VLcallback.c in H5VL_file_open(): line 3501: failed to iterate over available VOL connector plugins
        class: HDF5
        major: Virtual Object Layer
        minor: Iteration failed

    error #002: ../../src/H5PLpath.c in H5PL__path_table_iterate(): line 578: can't iterate over plugins in plugin path '(null)'
        class: HDF5
        major: Plugin for dynamically loaded library
        minor: Iteration failed

    error #003: ../../src/H5PLpath.c in H5PL__path_table_iterate_process_path(): line 712: can't open directory
        class: HDF5
        major: Plugin for dynamically loaded library
        minor: Can't open directory or file

    error #004: ../../src/H5VLcallback.c in H5VL__file_open(): line 3351: open failed
        class: HDF5
        major: Virtual Objec
Error in H5File.open(filename, mode, file_create_pl, file_access_pl) : 
  HDF5-API Errors:
    error #000: ../../src/H5F.c in H5Fopen(): line 620: unable to open file
        class: HDF5
        major: File accessibility
        minor: Unable to open file

    error #001: ../../src/H5VLcallback.c in H5VL_file_open(): line 3501: failed to iterate over available VOL connector plugins
        class: HDF5
        major: Virtual Object Layer
        minor: Iteration failed

    error #002: ../../src/H5PLpath.c in H5PL__path_table_iterate(): line 578: can't iterate over plugins in plugin path '(null)'
        class: HDF5
        major: Plugin for dynamically loaded library
        minor: Iteration failed

    error #003: ../../src/H5PLpath.c in H5PL__path_table_iterate_process_path(): line 712: can't open directory
        class: HDF5
        major: Plugin for dynamically loaded library
        minor: Can't open directory or file

    error #004: ../../src/H5VLcallback.c in H5VL__file_open(): line 3351: open failed
        class: HDF5
        major: Virtu

How can I solve this problem
